### ISSUE TITLE
ambiguous formulation

### ISSUE THREAD

**Issue by CJentzsch** [ethereum/yellowpaper#40]
You denote \sigma[a]_s to be the hash of the root node of the storage trie of an account (page 3). After defining a conveniant function which gives a new meaning to that variable you say "It shall be understood that σ[a]_s is not a ‘physical’ member of the account and does not contribute to its later
serialisation." You now have two meanings for the same variables, I think one should have two different names for it.

**Comment by gavofyork** [comment]
one 's' is bold, the other is not :)

**Comment by gavofyork** [comment]
I agree it's a bit of a pain to read though, this stuff will likely be fixed with a longer, more verbose version of the YP that I'll do soon.
